src/GraphQL/Utilities/TypeSettings.cs

```diff
@@ -74,7 +74,7 @@ public void Include(Type type)
         /// </summary>
         public void Include(string name, Type type)
         {
-            _typeConfigurations[name].Type = type;
+            For(name).Type = type;
```

Comment:
`ForAll` only triggers when using `For`, but seems to clearly indicate that it should operate on all types.  So I've changed `Include` to call `For` and thereby execute any configured `ForAll` code.